tests/HttpClientTest.php

```diff
@@ -79,7 +79,7 @@ public function testMassiveMetaDataHttpClient()
         $this->guzzle->expects($spy = $this->any())->method('post');
 
         // Add a report to the http and deliver it
-        $this->http->queue(Report::fromNamedError($this->config, 'Name')->setMetaData(['foo' => str_repeat('A', 1000000)]));
+        $this->http->queue(Report::fromNamedError($this->config, 'Name')->setMetaData(['foo' => str_repeat('A', HttpClient::MAX_SIZE * 2)]));
```

Comment:
This test (and the others) should still use a constant value to verify that the proper checks are happening and that `MAX_SIZE` has not been unexpectedly altered.
```suggestion
        $this->http->queue(Report::fromNamedError($this->config, 'Name')->setMetaData(['foo' => str_repeat('A', 1500000)]));
```